docs/flowcharts/high-level.md

```diff
@@ -17,15 +17,15 @@ graph TD;
   H -- Yes --> D
   H -- No --> I([Finish])
 
-  click B "./#find-a-reviewer" "Find a reviewer"
-  click C "./#meet-and-agree-on-objectives" "Meet and agree on objectives"
-  click D,E,F,G "./#perform-code-review" "Perform Code Review"
-  click I "./#finish" "Finish"
+  click B "./high-level.html#find-a-reviewer" "Find a reviewer"
+  click C "./high-level.html#meet-and-agree-on-objectives" "Meet and agree on objectives"
+  click D,E,F,G "./high-level.html#perform-code-review" "Perform Code Review"
+  click I "./high-level.html#finish" "Finish"
```

Comment:
We can actually get rid of any references to the current file and just use a relative reference within the current file

```suggestion
  click B "#find-a-reviewer" "Find a reviewer"
  click C "#meet-and-agree-on-objectives" "Meet and agree on objectives"
  click D,E,F,G "#perform-code-review" "Perform Code Review"
  click I "#finish" "Finish"
```